distribution/lambda/README.md

```diff
@@ -0,0 +1,58 @@
+
+# CDK template for running Quickwit on AWS Lambda
+
+## Python venv
+
+This project is set up like a standard Python project. The initialization
+process also creates a virtualenv within this project, stored under the `.venv`
+directory.  To create the virtualenv it assumes that there is a `python3`
+(or `python` for Windows) executable in your path with access to the `venv`
+package. If for any reason the automatic creation of the virtualenv fails,
+you can create the virtualenv manually.
+
+To manually create a virtualenv on MacOS and Linux:
+
+```bash
+python3 -m venv .venv
+```
+
+After the init process completes and the virtualenv is created, you can use the following
+step to activate your virtualenv.
+
+```bash
+source .venv/bin/activate
+```
+
+If you are a Windows platform, you would activate the virtualenv like this:
+
+```bash
+.venv\Scripts\activate.bat
+```
+
+Once the virtualenv is activated, you can install the required dependencies.
+
+```bash
+pip install -r requirements.txt
+pip install -r requirements-dev.txt
+```
+
+
+## Deploy and run
+
+The Makefile is a usefull entrypoint to show how the Lambda deployement can used.
+```bash
+export CDK_ACCOUNT=123456789
+export CDK_REGION=us-east-1
+make init
```

Comment:
Maybe we need to execute `make build` first?

I got this error

```
RuntimeError: Cannot find asset at /Users/fmassot/Documents/quickwit/repos/quickwit/quickwit/target/lambda/indexer
```

Reply:
I don't really understand why but looks like you do!